src/modules/75-cf/pages/feature-flags/FeatureFlagsPage.tsx

```diff
@@ -630,6 +162,19 @@ const FeatureFlagsPage: React.FC = () => {
   const onClearFilter = (): void => setFlagFilter({})
   const onClearSearch = (): void => searchRef.current.clear()
 
+  const onRowClick = (flagId: string): void => {
+    history.push(
+      withActiveEnvironment(
+        routes.toCFFeatureFlagsDetail({
+          orgIdentifier: orgIdentifier as string,
+          projectIdentifier: projectIdentifier as string,
+          featureFlagIdentifier: flagId,
+          accountId: accountIdentifier
+        })
+      )
+    )
+  }
```

Comment:
Not a blocker, but for callbacks like this, best to use `useCallback` to prevent unnecessary rerenders
```suggestion
  const onRowClick = useCallback((flagId: string): void => {
    history.push(
      withActiveEnvironment(
        routes.toCFFeatureFlagsDetail({
          orgIdentifier: orgIdentifier as string,
          projectIdentifier: projectIdentifier as string,
          featureFlagIdentifier: flagId,
          accountId: accountIdentifier
        })
      )
    )
  }, [withActiveEnvironment, orgIdentifier, projectIdentifier, accountIdentifier])
```